contracts/zapper-contract/SnowglobeZapperPangolin.sol

```diff
@@ -24,18 +24,8 @@ contract SnowglobeZapAvaxPangolin is ZapperBase {
         _removeLiquidity(address(pair), address(this));
 
         address swapToken = token1 == desiredToken ? token0 : token1;
-        address[] memory path = new address[](2);
-        path[0] = swapToken;
-        path[1] = desiredToken;
-
-        _approveTokenIfNeeded(path[0], address(router));
-        IPangolinRouter(router).swapExactTokensForTokens(
-            IERC20(swapToken).balanceOf(address(this)),
-            desiredTokenOutMin,
-            path,
-            address(this),
-            block.timestamp
-        );
+        _approveTokenIfNeeded(swapToken, address(router));
+        (IERC20[] memory path, ) = _swapToken(swapToken, desiredToken, IERC20(swapToken).balanceOf(address(this))); 
```

Comment:
is this our internal router?

Reply:
looks like it's the kyber router. Definitely a huge improvement over using a particular dex!

Are there reasons why we would want to use kyber over our internal router (YY fork)? If not, maybe worth moving to our router in the future.

Reply:
the axial router still needs some work (last time I tried). Probably will need to create a separate task for it.